src/iFixit/EventListener/iFixitApi.php

```diff
@@ -25,12 +26,18 @@ public function __construct(iFixitConfig $config, LoggerInterface $logger) {
       $this->config = $config;
       $this->logger = $logger;
 
+      $stack = HandlerStack::create();
+      $stack->push(Middleware::retry(function ($retries, $request, $response = null) {
+         return $retries <= 2;
+      });
```

Comment:
```suggestion
      }));
```

I guess we don't have a syntax check for CI on this repo ;-)